src/sentry/auth/idpmigration.py

```diff
@@ -74,6 +78,21 @@ def get_redis_key(verification_key: str) -> str:
     return f"auth:one-time-key:{verification_key}"
 
 
+def decode_verification_value(verification_value_byte):
+    return {
+        y.decode("ascii"): verification_value_byte.get(y).decode("ascii")
```

Comment:
Any reason to not use utf8 here?

Reply:
I also wonder if you used `client.setex` and `client.get` and simply stored the stringified json if you could avoid this decoding. (see src/sentry/utils/session_store.py)